tests/requirements.txt

```diff
@@ -18,4 +18,4 @@ nltk>=3.2
 # spacy>=2.0.18 
 pandas>=0.20
 umap-learn==0.3
-numba==0.42
\ No newline at end of file
+numba==0.42
```

Comment:
This must be something to do with whitespace? I'm not 100% sure what is causing this line to change.

Reply:
Newline was added in this PR. (Removed now)

Reply:
Good eye! Sorry, I wasn't trying to nitpick about it - but thank you for making the update! We'll get this merged in as soon as the CI completes.